modules/system/holoportos/udiskie.nix

```
@@ -0,0 +1,40 @@
+{ config, pkgs, ... }:
+let
+  wrapSystemdScript = (
+    program: ''
+      source ${config.system.build.setEnvironment}
+      exec ${program}
+    ''
+  );
+in rec {
+  environment.systemPackages = with pkgs; [
+    udiskie
+    udisks2
+  ];
+  services = {
+    udisks2.enable  = true;
+  };
+  systemd.user.services = {
+    udiskie = {
+      description = "Mounts disks in userspace with udisks";
+      serviceConfig = {
+        Type    = "simple";
+        Restart = "always";
+      };
+      script = wrapSystemdScript ''
+        ${pkgs.udiskie}/bin/udiskie \
+          -2                        \
+          --automount               \
+          --file-manager=""         \
+	  --no-notify               \
+      '';
+      wantedBy = [ "default.target" ];
+    };
+  };
+  users.extraUsers.user = {
+    extraGroups = [
+      "plugdev"
+    ];
+  };
```

Better syntax for all of that would be:

```suggestion
  users.users.user.extraGroups = [ "plugdev" ];
```